docker-compose/README.md

```diff
@@ -1,39 +1,61 @@
-# obds-to-fhir Docker Compose Version
+# diz-in-a-box Docker Compose Version
 
-## Run only the obds-to-fhir job
+![Figure Modular Pipeline](../img/fig1.png)
 
-```sh
-docker compose -f compose.obds-to-fhir.yaml up
-```
+## Installation
+
+### 1. Prepare Data Import
+#### a) ONKOSTAR data base connector
+Configure kafka-connect in [compose.full-yaml](compose.full.yaml) and [docker-compose/kafka-connect-passwords.properties](kafka-connect-passwords.properties).
+The oBDS single report XML-files will be loaded into the Kafka cluster.
+For more information about kafka-connect, refer to [7. Enable Kafka Connect and the connector.](#enable-kafka-konnect-and-the-connector)
```

Comment:
@chgl sollte dieser same page anchor nicht beim Klicken eig. zu 7. springen? tut er offenbar gerade nicht wenn ich das repo in diesem branch durchbrowse

Reply:
Ah, der anchor müsste: `#7-enable-kafka-connect-and-the-connector` heißen